package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "phantom-be-codebandits",
+	"version": "1.0.0",
+	"type": "module",
+	"description": "backend repository for phantom app",
+	"main": "app.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"prepare": "husky install",
+    "start":"node src/app.js",
+		"dev": "nodemon src/app.js",
+		"lint": "eslint --fix",
+		"format": "prettier --write \"./**/*.{js,jsx,json}\""
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/atlp-rwanda/phantom-be-codebandits.git"
+	},
+	"author": "codebandits",
+	"license": "ISC",
+	"bugs": {
+		"url": "https://github.com/atlp-rwanda/phantom-be-codebandits/issues"
+	},
+	"homepage": "https://github.com/atlp-rwanda/phantom-be-codebandits#readme",
```

Comment:
This entry is integreted different on github and is not useful for backend development

Reply:
bugs or homepage?